.circleci/config.yml

```diff
@@ -922,27 +922,45 @@ workflows:
                 at: /
           workflow-name: "nightly-performance-regression"
 
-  weekly-perf-start-from-saved-state-regression:
+  weekly-crypto-transfer-perf-start-from-saved-state-regression:
     triggers:
       - schedule:
-#  Make it run daily before converting to weekly
-#          cron: "0 1 * * 0,6"
-          cron: "0 10,22 * * *"
+          cron: "0 1,13 * * *"
           filters:
             branches:
               only:
                 - master
     jobs:
       - build-platform-and-services
-      - weekly-run-start-from-saved-state-regression:
+      - weekly-run-crypto-transfer-start-from-saved-state-regression:
           context: Slack
           requires:
             - build-platform-and-services
           pre-steps:
             - install-tools
             - attach_workspace:
                 at: /
-          workflow-name: "weekly-perf-start-from-saved-state-regression"
+          workflow-name: "weekly-crypto-transfer-perf-start-from-saved-state-regression"
+
+  weekly-HCS-perf-start-from-saved-state-regression:
```

Comment:
nit : HCS can also be in lower case

Reply:
Will change the names all together when changing the schedule from daily basis to weekly basis later.